apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -241,6 +242,20 @@ class ManageStudentsTable extends Component {
     );
   };
 
+  projectSharingFormatter = (projectSharing, {rowData}) => {
+    let disabled = rowData.isEditing ?
+      this.props.editingData[rowData.id].age.length === 0 :
```

Comment:
I realized age is sometimes a string "21+" and sometimes a number 5, does this length check work for both?